### PR TITLE
Merge develop 8.2 03.10.17

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

**Comment by livecodeali** [comment]
https://gist.github.com/42bfd0cff0fe65e485294ed31546b734

**Comment by livecodeali** [comment]
https://gist.github.com/af67c527ce0ea85c6da1c8eb8dbf6f8e

File: ide-support/revdeploylibraryios.livecodescript [hunk 1]
```diff
@@ -39,16 +39,23 @@ function deployGetIphoneOSes
...
    return tList
 end deployGetIphoneOSes
 
 function deployGetIosMinimumVersions
+<<<<<<< HEAD
    return "8.0,8.1,8.2,8.3,8.4,9.0,9.1,9.2,9.3,10.0,10.1,10.2,10.3"
+=======
+   return "6.0,6.1,7.0,7.1,8.0,8.1,8.2,8.3,8.4,9.0,9.1,9.2,9.3,10.0,10.1,10.2,10.3,11.0"
+>>>>>>> upstream/develop-8.2
 end deployGetIosMinimumVersions
```
Comment (livecodepanos) [inline ide-support/revdeploylibraryios.livecodescript]: Oops ^^^ :)

Comment (livecodeali) [reply ide-support/revdeploylibraryios.livecodescript]: oh damn :-) thanks

Comment (livecodeali) [reply ide-support/revdeploylibraryios.livecodescript]: Amended and updated the gist

**Comment by livecodepanos** [comment]
Looks fine to me.

Setting this to WIP as we need to rebuild the ICU and OpenSSL prebuilts for iOS 11 first, since they have different version in `develop`:

ICU: 58.2
OpenSSL: 1.1.0e

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok 28894a9

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok 28894a96758c076fb97c0c90c0703b8b1348bdbc

**Comment by livecode-vulcan** [comment]
😎 test success 28894a96758c076fb97c0c90c0703b8b1348bdbc

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/867)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/883)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/857)
* try-community-universal-ios-iphoneos11.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.0/builds/32)
* try-community-universal-ios-iphonesimulator11.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.0/builds/33)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/861)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/883)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/867)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/883)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/866)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/876)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/898)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/862)

**Pull request merged**